lib/pavilion/test_run.py

```diff
@@ -574,14 +576,18 @@ def get_permissions(pav_cfg, config) -> (str, int):
                     "exist on this system. {}"
                     .format(group, err))
 
-        umask = config.get('umask', pav_cfg['umask'])
+        umask = config.get('umask')
+        if umask is None:
+            umask = pav_cfg['umask']
         if umask is not None:
             try:
                 umask = int(umask, 8)
             except ValueError:
                 raise RuntimeError(
                     "Invalid umask. This should have been enforced by the "
```

Comment:
Not part of your changes, but this is a little repetitive.

Reply:
It's not. config['umask'] can have a value of `None` in that dictionary. It was a bug I ran across.